models/acronym.js

```diff
@@ -0,0 +1,16 @@
+const acronym = (sequelize, DataTypes) => {
+    constAcronym = sequelize.define('acronym', {
+        acronym: {
+            type: DataTypes.STRING
+        },
+        name: {
```

Comment:
Also, instead of `name`, what do you think about calling this `definition` or `meaning`? Calling it a name throws me off a little bit.

Reply:
I'm fine with that.